api/src/pcapi/alembic/versions/20250218T164119_060ef534e1a5_add_future_offer_reminders.py

```diff
@@ -0,0 +1,39 @@
+"""Add future offers reminders
+
+add_future_offer_reminders
+"""
```

Comment:
tu peux juste laisser l'un des deux 
```suggestion
"""Add future offers reminders
"""
```